app/src/ui/lib/conflicts/render-functions.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Octicon, OcticonSymbol } from '../../octicons'
+import { LinkButton } from '../link-button'
+
+export function renderUnmergedFilesSummary(conflictedFilesCount: number) {
+  // localization, it burns :vampire:
+  const message =
+    conflictedFilesCount === 1
+      ? `1 conflicted file`
+      : `${conflictedFilesCount} conflicted files`
+  return <h3 className="summary">{message}</h3>
+}
+
+export function renderAllResolved() {
```

Comment:
**Non-blocking**
I noticed the block below has `JSX.Element` in its signature, while the two above don't.

Reply:
Removed in f6c3c9705 as the type inference is Good Enough™